config/images.yml

```diff
@@ -221,85 +221,18 @@
     - "static/images/content/hero-images/physics_1383-mobile.jpg"
     - "static/images/content/hero-images/physics_1383-tablet.jpg"
 
-# blog
+# case studies
 "static/images/content/case-studies/abigail.jpg":
   alt: "Maths teacher Abigail in front of a maths display board."
 
 "static/images/content/case-studies/ben.jpg":
   alt: "Maths and science teacher Ben marking work at home."
 
-"static/images/content/blog/applying-for-teacher.jpg":
+# how to apply for teacher training
+"static/images/content/how-to-apply-for-teacher-training/notepad.jpg":
   alt: ""
   variants:
-    - "static/images/content/blog/thumbnails/applying-for-teacher.jpg"
```

Comment:
@emmaachesongray Just making sure you are aware that all these images that were being used in the blog are being deleted, in case any are needed for future (unlikely!)

@sarahcrack Can you just check with Emma before merging this PR (presumably not until w/c 6 Jan)?